packages/odyssey-design-tokens/src/hue.json

```diff
@@ -0,0 +1,167 @@
+{
+  "hue": {
```

Comment:
@gretchen-okta, @taylorridgway-okta and I thought this might make the limited use case and allowed values more clear.

`hue` is where we store the base values. `palette` is the token set that designers "paint" with. how does that jive with you?